RecoHGCal/TICL/plugins/PatternRecognitionbyCLUE3D.cc

```diff
@@ -474,8 +474,8 @@ void PatternRecognitionbyCLUE3D<TILES>::calculateLocalDensity(
       if (PatternRecognitionAlgoBaseT<TILES>::algo_verbosity_ > PatternRecognitionAlgoBaseT<TILES>::Advanced) {
         edm::LogVerbatim("PatternRecogntionbyCLUE3D") << "onSameLayer: " << onSameLayer;
       }
-      int etaWindow = onSameLayer ? 2 : 1;
-      int phiWindow = onSameLayer ? 2 : 1;
+      const int etaWindow = 2;
+      const int phiWindow = 2;
```

Comment:
Just for my knowledge, I most probably have missed some passages: why this was restricted fo `1` for `layerClusters` on different layers?

Reply:
Ciao @AdrianoDee, the original motivation for having 2 different cuts, within and outside of the layer, is related to the fact that we are clustering in 3D LayerClusters. Therefore there should be no need to further cluster them within every single layer or, if you want, you would need to search a little farther away to find some other clusters to collect on the same layer. The move to 2 was taken from the CA experience: we notice that, especially at high eta values, a single bin in `eta,phi` can be quite restrictive. Hence to move to just use one value, 2, to cover them all. When we reach the point of including `CLUE3D` in the reconstruction, we will need, possibly, some further tuning.